content/platform/hpe-machine-learning-development-environment/home-1.md

```diff
@@ -0,0 +1,37 @@
+---
+title: HPE Machine Learning Development Environment
+version: "1"
+description: HPE Machine Learning Development Environment helps developers train
+  machine learning models faster—in hours and minutes, not days and weeks—build
+  more accurate models, manage GPU costs,  and track and reproduce experiments.
+  Built upon the open source Determined Training Platform, this solution helps
+  you focus on innovation by removing the complexity and cost associated with
+  machine learning model development.
+image: /img/platforms/hpe-element.png
+width: large
+priority: 2
+active: false
+---
+Machine learning (ML) engineers and data scientists are on a never-ending search for new solutions that will enable them to better focus on innovation and accelerate their time to production—and this is what [HPE Machine Learning Development Environment](https://www.hpe.com/us/en/solutions/artificial-intelligence/machine-learning-development-environment.html) is all about.
+
+By removing the complexity and cost associated with ML model development, this comprehensive platform speeds time to value for model developers by:
+
+• Removing the need to write infrastructure code
+
+• Making it easier for IT administrators to set up, manage, secure, and share AI compute clusters
+
+With the HPE Machine Learning Development Environment, ML practitioners can:
+
+• Train models faster using state-of-the-art distributed training, without changing their model code
+
+• Automatically find high-quality models with advanced hyperparameter tuning from the creators of state-of-the-art tuning algorithms such as Hyperband
+
+• Get more from their GPUs with smart scheduling, as well as reduce cloud GPU costs by seamlessly using spot instances
+
+• Track and reproduce their work with experiment tracking that works out of the box, covering code versions, metrics, checkpoints, and hyperparameters
+
+Using a comprehensive array of features integrated into an easy-to-use, high-performance ML environment, ML engineers can focus on building better models, instead of managing IT infrastructure. Using the HPE Machine Learning Development Environment that supports both cloud and on-premises deployment infrastructure, practitioners can develop models using PyTorch, TensorFlow™, or Keras. HPE Machine Learning Development Environment also integrates seamlessly with today’s most popular ML tools for data preparation and model deployment.
+
+H﻿PE Machine Learning Development Environment is built upon the widely popular open source training platform, [Determined](determined.ai). You can see related articles about Determined on the HPE Developer portal [here](https://developer.hpe.com/platform/determined-ai/home). 
```

Comment:
Alex, please make sure to specify the full path URL for hotlink determined.ai
(https://www.determined.ai/)